src/docs/guides/rocket.md

```diff
@@ -0,0 +1,188 @@
+4. **Set Up a Public URL**:
+    - Navigate to the **Networking** section under the [Settings](/overview/the-basics#service-settings) tab of your new service.
+    - Click [Generate Domain](/guides/public-networking#railway-provided-domain) to create a public URL for your app.
+
+    **Note:** You'll come across a 502 error where your application doesn't respond. We'll fix that in the next step.
+5. **Configure Rocket app to accept non-local connections**:
+    - Rocket apps need to be configured to accept external connections by listening on the correct address, which is typically `0.0.0.0`. You can easily do this by setting the address through the environment variable.
+    Run the following command to set the Rocket address to `0.0.0.0`:
+        ```bash
+        railway variables --set "ROCKET_ADDRESS=0.0.0.0"
+        ```
+6. **Redeploy the Service**:
+    - Run `railway up` again to trigger a redeployment of the service.
+7. **Verify the Deployment**:
+    - Once the deployment completes, go to **View logs** to check if the server is running successfully. Access your public URL again and you should see your app working well.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/f_auto,q_auto/v1729858389/docs/quick-start/rocket_app_service.png"
+alt="screenshot of the deployed Rocket service"
+layout="responsive"
+width={2038} height={1698} quality={100} />
+
+### Deploy from a GitHub Repo
+
+To deploy a Rocket app to Railway directly from GitHub, follow the steps below:
+
+1. **Create a New Project on Railway**:
+    - Go to <a href="https://railway.app/new" target="_blank">Railway</a> to create a new project.
+2. **Deploy from GitHub**: 
+    - Select **Deploy from GitHub repo** and choose your repository.
+        - If your Railway account isn’t linked to GitHub yet, you’ll be prompted to do so.
+3. **Add Environment Variables**:
+    - Click **Add Variables**, then add **ROCKET_ADDRESS** with the value `0.0.0.0`. This allows your Rocket app to accept external connections by listening on `0.0.0.0`.
+3. **Deploy the App**: 
+    - Click **Deploy** to start the deployment process.
+    - Once the deployed, a Railway [service](/guides/services) will be created for your app, but it won’t be publicly accessible by default.
+4. **Verify the Deployment**:
+    - Once the deployment completes, go to **View logs** to check if the server is running successfully.
+
+    **Note:** During the deployment process, Railway will automatically [detect that it’s a Rust app](https://nixpacks.com/docs/providers/rust).
+5. **Set Up a Public URL**:
+    - Navigate to the **Networking** section under the [Settings](/overview/the-basics#service-settings) tab of your new service.
+    - Click [Generate Domain](/guides/public-networking#railway-provided-domain) to create a public URL for your app.
+
+### Use a Dockerfile
+
+1. Create a `Dockerfile` in the `helloworld` or Rocket app's root directory.
```

Comment:
I've always been told cargo chef is so much better, maybe we should show what a cargo chef Dockerfile looks like?

Reply:
Interesting. Let me check that instead.